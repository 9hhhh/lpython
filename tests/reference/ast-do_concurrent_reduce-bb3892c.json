{
    "basename": "ast-do_concurrent_reduce-bb3892c",
    "cmd": "lfortran --show-ast {infile} -o {outfile}",
    "infile": "tests/do_concurrent_reduce.f90",
    "infile_hash": "44ebdebb05b853f4680c203c3b2b0e98e9285d1f3923ecbc255bc105",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast-do_concurrent_reduce-bb3892c.stdout",
    "stdout_hash": "afc25e486644c4d30d3281498297fd995069a3be145590d3e9b7a0df",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}