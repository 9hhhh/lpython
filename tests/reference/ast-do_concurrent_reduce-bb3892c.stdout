(TranslationUnit [(Subroutine sum_reduce [(a) (s)] [] [(Declaration [(a "real" [(() ())] [(Attribute intent [(in)] [])] ())]) (Declaration [(s "real" [] [(Attribute intent [(out)] [])] ())]) (Declaration [(N "integer" [] [] ()) (i "integer" [] [] ())])] [(= N (FuncCallOrArray size [a] [])) (= s 0) (DoConcurrentLoop i 1 N () (Reduce reduce_opType0 [s]) [(= s (+ s (FuncCallOrArray a [i] [])))])] [])])
